policies/ucs_server_policies/network/san_connectivity_policy/terraform/san_connectivity_policy.tf

```diff
@@ -1,20 +1,36 @@
+provider "intersight" {
+  endpoint        = "https://intersight.com"
+  apikey          = "xxxxx27564612d30dxxxxx/5f21c9d97564612d30dd575a/5f9a8b877564612xxxxxxxx"
+  secretkey       = "C:\\secretKey.txt"
+}
+
+resource "intersight_server_profile" "server1" {
```

Comment:
do not create server profile in policy sample, it is already created as part of profile, remove it

Reply:
Removed.